### ISSUE TITLE
Inline conditionals not supported

### ISSUE THREAD

**Issue by aldesantis** [apotonick/erbse#4]
Erbse fails with inline conditionals such as:

```erb
<%= f.input :captain_hired if boat.captain_available? && !boat.captain_mandatory? %>
```

This will produce an obscure error like the following:

```
SyntaxError:
  app/concepts/boat_booker/views/show/_form.erb:5: syntax error, unexpected keyword_else, expecting keyword_end
  ; _buf << ("        </div>\n      ".freeze);  else ; 
                                                   ^
  app/concepts/boat_booker/views/show/_form.erb:12: syntax error, unexpected keyword_end, expecting end-of-input
  end;end;end;end
             ^
```

**Comment by george-carlin** [comment]
Possibly related to https://github.com/trailblazer/cells-erb/issues/5 ?

**Comment by Aethelflaed** [comment]
👍

**Comment by apotonick** [comment]
Fixed in 0.1.2.
